packages/api/src/routes/sensorRoute.js

```diff
@@ -44,8 +44,8 @@ router.get('/:location_id/reading', SensorController.getAllReadingsByLocationId)
 router.get('/reading/farm/:farm_id', SensorController.getReadingsByFarmId);
 router.post('/reading/invalidate', SensorController.invalidateReadings);
 router.post('/unclaim', SensorController.retireSensor);
-router.get('/:location_id/reading_type', SensorController.getSensorReadingTypes);
 router.get('/:farm_id/reading_type', SensorController.getAllSensorReadingTypes);
+router.get('/:location_id/reading_type', SensorController.getSensorReadingTypes);
```

Comment:
The issue is that we can't have two endpoints with the format `:id/reading_type` express isn't able to tell the difference. Please update to `/farm/:farm_id/reading_type` on both frontend and backend.

Reply:
That makes sense. I updated the endpoint both front and backend.